dev_tools/requirements/deps/dev-tools.txt

```diff
@@ -11,7 +11,8 @@
 asv
 
 # For verifying behavior of Quil output.
-pyquil~=2.21.0
+pyquil~=2.28.2; python_version < "3.7"
```

Comment:
I think we can actually remove these now that pyquil is pulled in by cirq-rigetti.